src/main/java/com/teamc/fms/controller/FacilityController.java

```diff
@@ -0,0 +1,44 @@
+package com.teamc.fms.controller;
+
+import com.teamc.fms.entity.Facility;
+import com.teamc.fms.service.FacilityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/facilities")
+public class FacilityController {
+
+    @Autowired
+    private FacilityService facilityService;
```

Comment:
Please use constructor injection and Lombok to avoid biolerplate code, like the attached image
![image](https://github.com/web-development-course-way/fms/assets/31379298/2ba3f3e5-2906-48e5-8ea2-c200ebb48c9b)